pyproject.toml

```diff
@@ -91,6 +91,13 @@ server = [
   "coloredlogs",
   "uuid6",
   "python-dateutil",
+  "cramjam",
```

Comment:
cramjam and packaging are consumer-only.

And why devtools is a production-level dependency?

Reply:
`from devtools import pformat`

`pformat` are used in setup of `DummyProvider` to output setting in logger.info.  I'am take this part from syncmaster.

Reply:
Look's OK for me

Reply:
Change to pformat from pprint